src/charts/loading/LoadingContainer.test.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import Enzyme, { mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-15';
+
+import LoadingContainer from './LoadingContainer';
+
+import barData from '../bar/barChart.fixtures';
+import bar from '../bar/barChart';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('Loading Container', () => {
+
+    it('should render without errors', () => {
+        expect(() => (
+            mount(
+                <LoadingContainer
+                    data={barData}
+                    loadingState={'.load-state'}
+                >
+                    <div className="chart" />
+                </LoadingContainer>
+            )
+        ).unmount()).not.toThrow();
+    });
+
+    describe('Loading States', () => {
+
+        describe('when data is not null', () => {
+            let wrapper;
+
+            beforeEach(() => {
+                wrapper = mount(
+                    <LoadingContainer
+                        data={barData}
+                        loadingState={'.load-state'}
+                    >
+                        <div className="chart" />
+                    </LoadingContainer>
+                );
+            });
+
+            it('should not show the loadingState after mounting', () => {
+                let expected = false;
+                let actual = wrapper.html().indexOf('load-state') > -1;
```

Comment:
would
   wrapper.find('.load-state').length
work?

Reply:
As discussed, class-based selectors were not working when searching for classes in the loading state markup.